src/blockdata/transaction.rs

```diff
@@ -239,6 +250,23 @@ pub enum SubField {
     MysteriousMinerGate(String),
 }
 
+impl Display for SubField {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), FmtError> {
+        match self {
+            SubField::TxPublicKey(PublicKey) => writeln!(fmt, "TxPublicKey: {}", PublicKey),
```

Comment:
IMO we should be consistent in the naming convention, we should use spaces and lowercase for the following words. E.g. `"Tx public key: {}"`

Reply:
Done